sdk/include/reactos/debug.h

```diff
@@ -58,6 +61,73 @@ RtlAssert(
     _In_opt_z_ PCHAR Message
 );
 
+#elif defined(_WIN32)
+/* Win32 User-Mode */
+
+#ifndef NO_STRSAFE
+#include <strsafe.h>
+#endif
+
+static inline
+ULONG
+__cdecl
+W32DbgPrint(_In_z_ _Printf_format_string_ PCSTR Format, ...)
+{
+    va_list va;
+    char szBuff[512];
```

Comment:
The idea of a user-adjustable `APPDBG_BUFSIZE` value (used here, and in the W32Assert below) sounded nice.

Reply:
`DEBUG_BUFSIZE`